src/components/views/groups/GroupMemberList.js

```diff
@@ -86,25 +87,28 @@ export default withMatrixClient(React.createClass({
         const GroupMemberTile = sdk.getComponent("groups.GroupMemberTile");
         query = (query || "").toLowerCase();
 
-        const memberList = this.state.members.filter((m) => {
-            if (query) {
+        let memberList = this.state.members;
+        if (query) {
+            memberList = memberList.filter((m) => {
                 //const matchesName = m.name.toLowerCase().indexOf(query) !== -1;
```

Comment:
Someone _might_ wonder why this is commented and it's not _super_ obvious that it's going to be implemented in future? Maybe a `TODO` ?

Reply:
fair :)